pinecone/client.go

```diff
@@ -504,6 +508,10 @@ func (req CreatePodIndexRequest) TotalCount() int {
 //		       fmt.Printf("Successfully created pod index: %s", idx.Name)
 //	    }
 func (c *Client) CreatePodIndex(ctx context.Context, in *CreatePodIndexRequest) (*Index, error) {
+	if in.Name == "" || in.Dimension == 0 || in.Metric == "" || in.Environment == "" || in.PodType == "" {
+		return nil, fmt.Errorf("name, dimension, metric, environment, and podtype must be included in CreatePodIndexRequest")
```

Comment:
I'd capitalize the keys for the variables in this string as well.

Reply:
Ah okay! My IDE keeps telling me that `Errorf` stmts can't start with a capital letter, so I kept 'em lowercased

Reply:
Ah you're right, mine does the same I think it's a Go thing. 🤔 

Might have to lead with a different word if that's the case.